packages/client/src/utils/referenceApi.ts

```diff
@@ -22,6 +22,17 @@ export interface ILocationDataResponse {
 export interface IFacilitiesDataResponse {
   [facilityId: string]: ILocation
 }
+
+type FontFamilyTypes = {
+  normal: string
+  bold: string
+  italics: string
+  bolditalics: string
```

Comment:
@jpye-finch what do you think of this.  Essentially we are restricting the possibilities to 4 types of font.  https://fonts.google.com/knowledge/glossary/family_or_type_family_or_font_family

Reply:
Yes not sure if required to be honest. 

Just need to allow them to say what fonts they are using in the certificate. Is there any reason we need to restrict the numbers. 

An example below. ie you might have two bolds 

Font 1: Roboto Bold
Font 2: Noto Sans Bold
Font 3: Roboto Medium
Font 4: Roboto Light

Reply:
We are not restricting a country from supplying multiple fonts but its just that our pdf renderer library only supports 4 variants of a single font: Regular, Bold, Italic & BoldItalic.

Reply:
But @jpye-finch mentioned if we want more weights, we can add multiple fonts like Roboto Light, Roboto Medium

Reply:
The above examples don't have variant fonts. 

**Roboto** is the font family. 

Then from Google Fonts, I can choose to download the individual font variants. Eg. Roboto Bold, Roboto Light

Reply:
The country should just need to upload the font variants they use imo